aks/application/resources.tf

```
@@ -195,3 +195,19 @@ resource "kubernetes_ingress_v1" "main" {
     }
   }
 }
+resource "kubernetes_pod_disruption_budget_v1" "pdb" {
+  count = var.replicas > 1 ? 1 : 0
+
+  metadata {
+    name      = "${local.app_name}-pdb"
```

```suggestion
    name      = local.app_name
```

Do we need the suffix? Since we already know this object is a pod disruption budget type, I think it's superfluous.

---

Agreed and changed